test/jdk/java/util/TimeZone/TimeZoneData/displaynames.txt

```diff
@@ -163,11 +163,9 @@ Europe/Simferopol MSK
 Europe/Sofia EET EEST
 Europe/Tallinn EET EEST
 Europe/Tirane CET CEST
-Europe/Uzhgorod EET EEST
 Europe/Vienna CET CEST
 Europe/Vilnius EET EEST
 Europe/Warsaw CET CEST
-Europe/Zaporozhye EET EEST
```

Comment:
Do we need to remove those newly linked zone ids from testing?

Reply:
This file is not maintained by hand but generated from java/util/TimeZone/TimeZoneData/tools/share/Makefile.  
These two Ukraine zones are integrated to Europe/Kyiv and removed from tzdata/europe, so the entries disappeared in this file.